diag_manager/include/fms_diag_fieldbuff_update.inc

```diff
@@ -0,0 +1,31 @@
+#undef FMS_DIAG_FBU_DATA_TYPE_
```

Comment:
@mlee03 can you look at this and make sure it's following your standards.

Reply:
@thomas-robinson @ngs333, instead of TYPE we've been using KIND macros, e.g., #define FMS_DIAG_FBU_DATA_KIND_ r4_kind

Reply:
We've been using "kind" macros instead of "type", e.g, `FMS_DIAG_FBU_DATA_KIND_ r4_kind`.  Also we put the macro definitions in the _r4.fh and _r8.fh files (separate files for r4 and r8_kind); the code went into the *.inc files.